editoast/src/views/timetable/paced_train.rs

```diff
@@ -482,6 +486,70 @@ async fn project_path(
     Ok(Json(project_path_result))
 }
 
+/// Computes the occupancy blocks of a given path for a set of paced trains.
+///
+/// - Returns 404 if the infra or any of the paced trains are not found
+/// - Returns 200 with a hashmap of train_id to OccupancyBlock
+#[utoipa::path(
+    post, path = "",
+    tag = "paced_train",
+    request_body = OccupancyBlockForm,
+    responses(
+        (status = 200, description = "Occupancy Block Output", body = HashMap<i64, OccupancyBlockResult>),
```

Comment:
```suggestion
        (status = 200, body = HashMap<i64, OccupancyBlockResult>),
```

Obvious from context, either describe what the output means or no need to bother

Reply:
done